tests/unit/data_formats/test_roboflow.py

```diff
@@ -7,7 +7,7 @@
 import numpy as np
 import pytest
 
-from datumaro.components.annotation import Bbox, Label, Polygon
+from datumaro.components.annotation import Bbox, Label, Polygon, RotatedBbox
```

Comment:
Please remove `Polygon` import as it is not accessed anymore

Reply:
fixed at 2bbfbaeee81b4c8f89a67ccc937dce9c7c7d4aef